### PR TITLE
CLOUD-815 Fix PGO Jenkins pipeline which is being run upon PR

### PR DESCRIPTION
**Pull request by ptankov**
[CLOUD-815](https://jira.percona.com/browse/CLOUD-815) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are the manifests (crd/bundle) regenerated if needed?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PG version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/demand-backup.log) |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/init-deploy.log) |
| monitoring | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/monitoring.log) |
| operator-self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/operator-self-healing.log) |
| scaling | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/scaling.log) |
| scheduled-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/scheduled-backup.log) |
| self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/self-healing.log) |
| start-from-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/start-from-backup.log) |
| telemetry-transfer | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/telemetry-transfer.log) |
| users | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/238ea8387/users.log) |
| We run 0 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/238ea8387aa6abd7c1f385651909a52332976265
image: `perconalab/percona-postgresql-operator:PR-552-238ea8387`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/demand-backup.log) |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/init-deploy.log) |
| monitoring | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/monitoring.log) |
| operator-self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/operator-self-healing.log) |
| scaling | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/scaling.log) |
| scheduled-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/scheduled-backup.log) |
| self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/self-healing.log) |
| start-from-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/start-from-backup.log) |
| telemetry-transfer | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/telemetry-transfer.log) |
| users | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/8689e0254/users.log) |
| We run 0 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/8689e025484798fdb3ea27e0e8aa472b7a7cdaf0
image: `perconalab/percona-postgresql-operator:PR-552-8689e0254`

File: Jenkinsfile [hunk 1]
```diff
@@ -225,40 +222,34 @@ void runTest(Integer TEST_ID) {
...
-
-        # v0.15.0 kuttl version
-        kubectl krew install --manifest-url https://raw.githubusercontent.com/kubernetes-sigs/krew-index/a67f31ecb2e62f15149ca66d096357050f07b77d/plugins/kuttl.yaml
-        printf "%s is installed" "$(kubectl kuttl --version)"
-        kubectl krew install assert
-    '''
+        curl -fsSL https://github.com/kubernetes-sigs/krew/releases/latest/download/krew-linux_amd64.tar.gz | tar -xzf -
+        ./krew-linux_amd64 install krew
+        export PATH="\${KREW_ROOT:-\$HOME/.krew}/bin:\$PATH"
+        kubectl krew install kuttl assert
```
Comment (hors) [inline Jenkinsfile:240]: As I remember, we had some problems with the latest kuttl and decided to freeze it. @ptankov do we have any problems now?

Comment (ptankov) [reply Jenkinsfile:240]: I don't have any knowledge about problems with kuttl.

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/e5164c252/init-deploy.log) |
| We run 0 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/e5164c252d1f238754c16d46c2b1f0e89c23dc17
image: `perconalab/percona-postgresql-operator:PR-552-e5164c252`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/31e91c967/init-deploy.log) |
| We run 0 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/31e91c9671fb01045ce9f973585cc85fe66c9444
image: `perconalab/percona-postgresql-operator:PR-552-31e91c967`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/1eaaa37c3/init-deploy.log) |
| We run 0 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/1eaaa37c3b53caf7d52745969d3d5ac9fe03b521
image: `perconalab/percona-postgresql-operator:PR-552-1eaaa37c3`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/68e38e07a/init-deploy.log) |
| We run 0 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/68e38e07a4096f539b7078644a5bf01afeb27bb0
image: `perconalab/percona-postgresql-operator:PR-552-68e38e07a`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/92a504137/init-deploy.log) |
| We run 1 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/92a504137820e5aa373650735df5666f45a3f6c3
image: `perconalab/percona-postgresql-operator:PR-552-92a504137`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/demand-backup.log) |
| init-deploy | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/init-deploy.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/monitoring.log) |
| operator-self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/operator-self-healing.log) |
| scaling | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/scaling.log) |
| scheduled-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/scheduled-backup.log) |
| self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/self-healing.log) |
| start-from-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/start-from-backup.log) |
| telemetry-transfer | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/telemetry-transfer.log) |
| users | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/cd5124e43/users.log) |
| We run 4 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/cd5124e434ad1ba25773f2452b2d81c685e931fb
image: `perconalab/percona-postgresql-operator:PR-552-cd5124e43`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/init-deploy.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/monitoring.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/operator-self-healing.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/start-from-backup.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/telemetry-transfer.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/0b0d5b304/users.log) |
| We run 10 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/0b0d5b30484e9a66a0b86dad66e8ee4fa207c2b8
image: `perconalab/percona-postgresql-operator:PR-552-0b0d5b304`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/demand-backup.log) |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/init-deploy.log) |
| monitoring | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/monitoring.log) |
| operator-self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/operator-self-healing.log) |
| scaling | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/scaling.log) |
| scheduled-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/scheduled-backup.log) |
| self-healing | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/self-healing.log) |
| start-from-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/start-from-backup.log) |
| telemetry-transfer | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/telemetry-transfer.log) |
| users | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/61ace8019/users.log) |
| We run 0 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/61ace8019f3a20e69cb07d85303c22ac5b10d6ee
image: `perconalab/percona-postgresql-operator:PR-552-61ace8019`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/f798e5a8a/init-deploy.log) |
| We run 1 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/f798e5a8a0a0af2b0f76be2b6f2e2800b6ee8147
image: `perconalab/percona-postgresql-operator:PR-552-f798e5a8a`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/d7ba484ce/init-deploy.log) |
| We run 1 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/d7ba484cef5858c51b6a0f3f04ef883ba9869718
image: `perconalab/percona-postgresql-operator:PR-552-d7ba484ce`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/init-deploy.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/monitoring.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/operator-self-healing.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/start-from-backup.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/telemetry-transfer.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/3a7d2dede/users.log) |
| We run 10 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/3a7d2dedeb2f0ec80f920c23c4883b8450821563
image: `perconalab/percona-postgresql-operator:PR-552-3a7d2dede`

**Review by inelpandzic — APPROVED** [review]

**Review by egegunes — DISMISSED** [review]

**Review by hors — APPROVED** [review]

**Review by tplavcic — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/init-deploy.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/monitoring.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/operator-self-healing.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/start-from-backup.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/telemetry-transfer.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-552/85c31a265/users.log) |
| We run 10 out of 10|

commit: https://github.com/percona/percona-postgresql-operator/pull/552/commits/85c31a2650bfa8da870d498ad37e279bcc75f562
image: `perconalab/percona-postgresql-operator:PR-552-85c31a265`

**Review by hors — APPROVED** [review]

**Pull request merged**